### DISCUSSION TITLE
Command line interfaces

### DISCUSSION THREAD

**Discussion by brettin** [ShishirPatil/gorilla#53] [Q&A] [answered]
I have an extensive command line interface containing over 100 commands. Is it possible to teach the model a command line interface instead of a traditional API?

**Comment by tianjunz** [comment, marked as answer]
Hey @brettin, thanks for raising up the question! We actually have a new command line interface to support Linux commands. It is as simple as pip install gorilla-cli, and then do go $user_query. Give it a try and let us know what you think! 🚀
